src/content/docs/distributed-tracing/concepts/introduction-distributed-tracing.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Introduction to distributed tracing
+title: Track requests across your microservices
```

Comment:
I do think we need to get distributed tracing in here somehow, although I do think this is a better title.

How about: "Distributed tracing: Track requests across your microservices"?